Gemfile

```diff
@@ -34,9 +34,6 @@ end
 
 group :development do
   gem 'listen', '>= 3.0.5', '< 3.2'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
```

Comment:
Removed due to https://stackoverflow.com/questions/23165506/rails-spring-breaking-generators